src/lib/bookmarks/bookmarkstoolbar.cpp

```diff
@@ -123,17 +132,17 @@ void BookmarksToolbar::showOnlyIconsChanged(bool state)
     }
 }
 
-void BookmarksToolbar::openBookmarkInNewTab()
```

Comment:
This doesn't look right, you removed `openBookmarkInNewTab` and `openBookmarkInNewWindow`.